backend/sozisel/test/support/factory.ex

```diff
@@ -64,7 +66,7 @@ defmodule Sozisel.Factory do
       name: attrs[:event_name] || sequence(:event_name, &"some-event-#{&1}"),
       session_template_id: attrs[:session_template_id],
       start_minute: attrs[:start_minute] || 2137,
-      event_data: build(:event_data, type: attrs[:type] || :quiz)
+      event_data: attrs[:event_data] || build(:event_data, type: attrs[:type] || :quiz)
```

Comment:
It is possible to get not only quiz but random with all types?

Reply:
I don't like the random idea. I would rather make this a mandatory option rather than putting there a quiz by default if we want to change anything.

Reply:
I will refactor those `event_factory` to be separate for each event, therefore having explicit `quiz_event_factory`, `poll_event_factory` etc.